benchmarks/GeneratedHydratorBenchmark/AllPrivateClass.php

```diff
@@ -1,16 +1,26 @@
 <?php
 
+declare(strict_types=1);
+
 namespace GeneratedHydratorBenchmark;
 
 /**
  * Class that contains only private properties
  */
 class AllPrivateClass
 {
+//phpcs:disable SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedProperty
```

Comment:
I'd say it's preferable to do file-level excludes in phpcs.xml.dist rather than using inline comments.